tests/multipleBackend/multipartUpload.js

```diff
@@ -235,6 +243,32 @@ describe('Multipart Upload API with AWS Backend', function mpuTestSuite() {
         });
     });
 
+    it('should abort a multipart upload on real AWS', done => {
+        mpuSetup(uploadId => {
+            deleteParams.url = `/${objectKey}?uploadId=${uploadId}`;
+            deleteParams.query = { uploadId };
+            multipartDelete(authInfo, deleteParams, log, err => {
+                assert.equal(err, null, `Error aborting MPU: ${err}`);
+                s3.listParts({ Bucket: awsBucket, Key: objectKey,
+                UploadId: uploadId }, err => {
+                    assert.strictEqual(err.code, 'NoSuchUpload');
+                    done();
+                });
+            });
+        });
+    });
+
```

Comment:
except if it is in a region where an error is returned by aws, they would return NoSuchUpload and we would translate that to InternalError which is not actually bad behavior since they caused a diversion in state on scality s3 and aws s3 by aborting directly on aws.